languages/javascript/concepts/errors/about.md

```diff
@@ -1 +1,67 @@
-TODO: add information on errors concept
+Errors are useful to report when something is wrong or unexpected in a program or a piece of code.
+
+They are javascript objects.
+
+The main property of this object is `message`:
+
+```javascript
+const error = new Error('Oops, something went wrong')
+
+console.log(error.message)
+// => "Oops, something went wrong"
+```
+
+Using the `throw` syntax, you can throw an Error.
+
+```javascript
+throw new Error('Oops')
+```
+
+When an Error is thrown, the current execution is stopped and resume in the first catch block of the call stack.
```

Comment:
seems to me that `resume` would be better being plural

Reply:
aaa7fe2a557b19ad60f0a2626d6e2f7b734d1f0c